wiki/FitNesseRoot/HsacExamples/SlimTests/BrowserTests.wiki

```diff
@@ -68,6 +72,7 @@ The [[heuristic to find places][>FindingByHeuristics]] tries a number of transla
 |scroll to <place>                                                         |scrolls the designated place into view                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
 |hover over <place>                                                        |simulates hovering over the designated place with the mouse (so e.g. 'mouseover' behavior can be tested)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
 |drag and drop <place> to <target>                                         |drags the designated place to the target (also works for HTML5 drag and drop if the source element has the !style_code[draggable] attribute)                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+|drag and drop <element> at xoffset <xoffset> and yoffset <yoffset>        |drags the designated element to the target offset from current location                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
Can we rename this to something like `drag and drop <place> to offset X <horizontalOffset> Y <horizontalOffset>`. Using 'at' seems wrong here as the offset is where we drag to, correct?

Reply:
Changes done.